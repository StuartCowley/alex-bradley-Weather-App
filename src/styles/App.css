.weather-app{
    margin: 0 20px;
    display: flex;
    flex-direction: column;
}
.weather-app > * {
    padding: 20px 0px;
}