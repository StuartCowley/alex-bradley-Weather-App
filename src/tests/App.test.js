import { render } from "@testing-library/react";
import React from "react";
import App from "../components/App";

describe("App", () => {
  it("renders learn react link", () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
