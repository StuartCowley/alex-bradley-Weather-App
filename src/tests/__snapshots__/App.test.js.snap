// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders learn react link 1`] = `
<DocumentFragment>
  <div
    class="weather-app"
  >
    <h1
      class="location-details"
    >
      , 
    </h1>
    <form
      class="search-form"
    >
      <input
        class="search-form__input"
        placeholder="Search City"
        type="text"
        value=""
      />
      <button
        class="search-form__button"
        data-testid="search-form__button"
        type="submit"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-magnifying-glass "
          data-icon="magnifying-glass"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M416 208c0 45.9-14.9 88.3-40 122.7L502.6 457.4c12.5 12.5 12.5 32.8 0 45.3s-32.8 12.5-45.3 0L330.7 376c-34.4 25.2-76.8 40-122.7 40C93.1 416 0 322.9 0 208S93.1 0 208 0S416 93.1 416 208zM208 352c79.5 0 144-64.5 144-144s-64.5-144-144-144S64 128.5 64 208s64.5 144 144 144z"
            fill="currentColor"
          />
        </svg>
      </button>
    </form>
    <div
      class="forecast-summaries"
    />
  </div>
</DocumentFragment>
`;
